src/modules/party-external-rating/party-external-rating.controller.test.ts

```diff
@@ -0,0 +1,105 @@
+import { RandomValueGenerator } from '@ukef-test/support/random-value-generator';
```

Comment:
I'm probably missing something -- but might it be clearer to have these test files within the top-level `test/` directory, instead of the same directory alongside the controller? I'd just expect when the migrated API grows, it'd be easier for developers to navigate if we had a clean split between application and test code.

Reply:
@IgnatG and I discussed this [here](https://github.com/UK-Export-Finance/tfs-api/pull/8#discussion_r1116263235)

Reply:
Oh, sorry 😅

Reply:
That's alright, I'm sorry I forgot to mention it to everyone :)